db/migrate/20210409124256_add_urn_to_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddUrnToSite < ActiveRecord::Migration[6.1]
+  def change
+    add_column :site, :urn, :string
```

Comment:
Are duplicate values allowed? ie could a provider set the same urn for different sites?

Reply:
Spoke to Conor offline. Duplicate URN’s and UKPR’s are permitted